### DISCUSSION TITLE
Question regarding copyright

### DISCUSSION THREAD

**Discussion by nathan-a-macleod** [domlysz/BlenderGIS#507] [General]
# **Blender and OS versions**
Not applicable.

# **Describe the bug**
So I have a question regarding BlenderGIS. I'm not 100% sure if this is the right place to put this as it is more of a question than a bug - so please feel free to let me know if there is somewhere else I should put this.
But anyway, my question is: if I were to use BlenderGIS to create some 3d terrain of an area, then to export it to some format such as FBX, OBJ, etc, would I be able to use that in some kind of game using Godot, Three.js, or something else? I mean in terms of copyright - would I need to give attribution to BlenderGIS or someone else or is BlenderGIS satellite & map data open source? The game would most likely be free & open source.
Many thanks.

# **How to Reproduce**
Not applicable.

# **Error message**
Not applicable.

**Comment by domlysz** [comment]
You must mention the copyright of the datasources not the addon ! Futhermore you must check each datasource license to make sure that you do not violate the terms of use. SRTM (relief) and OpenStreeMap data uses friendly licences, but Google or Bing aerial photos cannot be used for free. In fact the addon itself infringe copyright of these sources by allowing the download. It's not a big deal for a small personnal project by it should be avoiding in the context of a commercial project.
